# Order is stacked in reverse so that when viewing in terminal, interesting 
# data is outputted last (don't have to scroll up past an image and dvc!)
uint8[] queryImage  # image grabbed by publisher
std_msgs/Header header
aarapsi_robot_pack/LabelDetails data
